dotnet/src/SemanticKernel.Abstractions/Orchestration/ContextVariables.cs

```diff
@@ -17,26 +19,28 @@ public sealed class ContextVariables : IEnumerable<KeyValuePair<string, string>>
     /// <summary>
     /// In the simplest scenario, the data is an input string, stored here.
     /// </summary>
-    public string Input => this._variables[MainKey];
+    public string Input => this._variables[MainKey].Value;
 
     /// <summary>
     /// Constructor for context variables.
     /// </summary>
     /// <param name="content">Optional value for the main variable of the context.</param>
-    public ContextVariables(string? content = null)
+    /// <param name="isTrusted">Whether the content is trusted or not (default true).</param>
+    public ContextVariables(string? content = null, bool isTrusted = true)
     {
-        this._variables[MainKey] = content ?? string.Empty;
+        this._variables[MainKey] = new TrustAwareString(content ?? string.Empty, isTrusted);
```

Comment:
(don't block on this):
For a future PR: we can either just allow null to flow through to the ctor, and/or create a TrustAwareString.Empty static instance (which should probably just be isTrusted=true, cause it's empty.)

Reply:
yeah, there's a `TrustAwareString.Empty` but we'll have to check how nullability propagates through the code